src/main/java/net/smartcosmos/dao/relationships/impl/RelationshipPersistenceService.java

```diff
@@ -327,6 +422,12 @@ private RelationshipEntity persist(RelationshipEntity relationshipEntity) throws
         }
     }
 
+    /**
+     * Converts a list of RelationshipEntity to a list of RelationshipResonse.
+     *
+     * @param entityList
+     * @return
+     */
     private List<RelationshipResponse> convertList(List<RelationshipEntity> entityList) {
```

Comment:
I think this would be a `Converter<List<RelationshipEntity>, List<RelationshipResponse>>` converter which a aware of the conversion service.

Thus, the code below effectively moved to the `convert()` method in the new converter class.